includes/blast_ui.services.inc

```diff
@@ -640,375 +640,6 @@ function getBlastData( $job_id )
    return $result;
  }
 
-/*
```

Comment:
function moved to blast_ui.form_advanced_options.inc because this file will be replaced with a different web services implementation.